store_test.js

```diff
@@ -32,26 +34,49 @@ xScenario('create account', ({ I, homePage, authPage, createAccountPage, myAccou
 }).tag('reg');
 
 
-Scenario('buy product', async ({ I, homePage, authPage, myAccountPage, productPage }) => {
+Scenario('buy product', async ({ I, homePage, authPage, myAccountPage, productPage, cartPage }) => {
     homePage.clickSignIn();
     authPage.login('1661344566222@test.com', '123456789A');
     myAccountPage.verifyAccountHeader();
     I.amOnPage('http://automationpractice.com/index.php?id_product=2&controller=product');
     let productPrice = await productPage.getProductPrice();
+    I.getNumericPrice(productPrice);
```

Comment:
result of `I.getNumericPrice()` is not used(not assigned to variable or used afterwards). That is why `I.assertEqual()` fails